README.md

```diff
@@ -24,50 +24,53 @@ Recurse Roulette requires users to authenticate with Recurse Center before signi
 
 ### Server Code
 
-When a client connects to the server, that client is given a peer ID. The server keeps track of the peer IDs as they connect. When a client requests to connect to another client, the server returns a list of available peers to connect to and the client randomly picks an id from this list. Once two peers are connected, both peer ids are removed from this list. The server also keeps track of all online clients. When a client ends a connection with another client, their peer id is destroyed and they are taken. To rejoin, they reconnect and are given a new peer id.
-The sever exposes three endpoints for the client to use:
+Recurse Roulette shuffles and reconnects clients every five minutes. To keep the clock consistent across all clients, the server keeps track of the time until the next chat round. Clients request the time from the server as soon as they connect.
+
+Clients need to click "Join" to connect to the signaling server and to be paired for the next chat round. When this happens, the client is given a peer ID by the server. When a new round starts, each client that has joined requests a partner from the server. The server pairs each client as the requests come in. If there is another client that has requested a partner, the server pairs those two clients, otherwise the server remembers the client’s peer ID until another client requests a partner. When the round ends, each client's peer ID is destroyed, which ends the current call, and each client is given a new ID for the next round. If a client disconnects, its peer ID is destroyed, and it must rejoin to receive a new one.
+
+The sever exposes three endpoints:
```

Comment:
```suggestion
The server exposes three endpoints:
```

Reply:
sever is what I meant!

Reply:
You're right. A sever would probably expose at least three endpoints. A severe sever? Possibly more!